src/span_context.js

```diff
@@ -149,6 +111,13 @@ export default class SpanContext {
         ].join(':');
     }
 
+    withBaggageItem(key: string, value: string): SpanContext {
+        let newBaggage = _.assign(this._baggage);
```

Comment:
according to [this,](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/assign) `assign()`'s first argument is the target of a copy.